triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
-+-------------------+------------------------------------------------------+
-| Extension Modules | C modules in Modules_.                               |
-+-------------------+------------------------------------------------------+
-| Interpreter Core  | The interpreter core.                                |
-|                   | The built-in objects in `Objects`_, the `Python`_,   |
-|                   | `Grammar`_ and `Parser`_ dirs.                       |
-+-------------------+------------------------------------------------------+
-| Library (Lib)     | Python modules in Lib_.                              |
-+-------------------+------------------------------------------------------+
-| Tests             | The unittest framework in `Lib/unittest`_            |
-|                   | The doctest framework `Lib/doctest.py`_.             |
-|                   | The CPython tests in `Lib/test`_.                    |
-|                   | The test runner in `Lib/test/regrtest.py`_.          |
-|                   | The test support utilities in `Lib/test/support`_.   |
-+-------------------+------------------------------------------------------+
-
-Versions
---------
-The known versions of Python that the issue affects and should be fixed for.
-
-Thus if an issue for a new feature is assigned for e.g., Python 3.8 but is not
-applied before Python 3.8.0 is released, this label should be updated to say
-``python-3.9`` as the version and drop ``python-3.8``.
-
-Priority
---------
-What is the severity and urgency?
-
-+------------------+--------------------------------------------------------+
-| Priority         | Description                                            |
-+==================+========================================================+
-| normal           | The default value for most issues filed.               |
-+------------------+--------------------------------------------------------+
-| deferred blocker | The issue will not hold up the next release, *n*. It   |
-|                  | will be promoted to a *release blocker* for the        |
-|                  | following release, *n+1*.                              |
-+------------------+--------------------------------------------------------+
-| release blocker  | The issue **must** be fixed before *any* release is    |
-|                  | made, e.g., will block the next release even if it is  |
-|                  | an alpha release.                                      |
-+------------------+--------------------------------------------------------+
-
-As a guideline, whether a bug is a *release blocker* for the current
-:ref:`release schedule <branchstatus>` is decided by the release manager.
-Triagers may recommend this priority and should notify the release manager by
-tagging them in a comment using ``@username``. If needed, consult the
-:ref:`release schedule <branchstatus>` and the release's associated PEP for the
-release manager's name.
```

Comment:
This was also adapted from BPO and it's now covered in the `Other labels` section.

Reply:
It would seem to be very helpful to port over the key points of the paragraph at the bottom, though, to make clear that the RM generally sets these labels, not triagers (or even core devs)? I recall this being a point of confusion at least a couple times in the past, when a triager set this label not knowing that it was the RM's call instead. I've added a suggestion above implementing this.

Reply:
I was under the impression that triagers could set these labels to bring the issue up to the attention of the RMs if they think it's a blocker.  Since the RMs should be subscribed to the labels and follow the "Release and deferred blockers" project, they should get notified and can then decide whether it was a good call or not, and possibly remove the label.

This also solves the issue of having to figure out who the RMs are, pinging them manually, and risking that the ping gets lost among other notifications.

Perhaps we should ask the RMs if this workflows is preferable or if they prefer to be pinged manually as the removed docs suggests.

P.S. apparently we have a @python/release-managers-in-development-maintenance-and-security-mode team that could be pinged instead of looking up the RM name.

Reply:
That's a great point. The current text implies (but does not explicitly state) that triagers should only _ping_ the RM in a comment _recommending_ this priority (with no explicit guidance for core devs), while the RM should apply it, and I was also led to believe the same by one of Pablo's Discord comments in response to a triager proposing a release blocker on there, that "all release blockers should be validated by the RM", which [my previous suggestion](https://github.com/python/devguide/pull/930#discussion_r1056997730) reflects.

However, the does sound like a potentially simpler and more efficient workflow for triagers and RMs alike. However, it relies on triagers being able to have a reasonable idea of what might qualify as a release blocker, which in turn requires us to provide reasonably clear and specific guidance to them here, as well as the RMs being comfortable with that. In that case, I've made [a followup suggestion](https://github.com/python/devguide/pull/930#discussion_r1057294582) implementing this alternative instead.